pipelines/test/tw/contains-files.yaml

```diff
@@ -0,0 +1,89 @@
+name: contains-files
+
+inputs:
+  dir:
+    description: |
+      Directory to look into. Defaults to /usr.
+      Examples:
+        dir: "/usr/lib/"
+    required: false
+    default: "/usr/"
+  name:
+    description: |
+      A find-like name expression. Defaults to * (i.e, match anything).
+    required: false
+    default: "*"
+  type:
+    description: |
+      File type to find. Defaults to 'f' (i.e, regular file).
+    required: false
+    default: "f"
+  files:
+    description: |
+      Space-separated list of file paths to check for existence.
+      If present, parameters 'dir', 'name' and 'type' will be ignored.
+    required: false
+    default: ""
+
+# USAGE EXAMPLES:
+#
+#   - uses: test/tw/contains-files
+#     with:
+#       dir: /usr/lib/
+#       name: libre2.so
+#
+#   - uses: test/tw/contains-files
+#     with:
+#       dir: /opt/
+#       name: some-binary-link
+#       type: l
+#
+#    - uses: test/tw/contains-files
+#      with:
+#        dir: "/usr/lib/ruby/gems/"
+#        name: "*.rbs"
+#    - uses: test/tw/contains-files
+#      with:
+#        files: |
+#          /usr/lib/ruby/gems/3.3.0/gems/webrick-1.9.2/Gemfile
+#          /usr/lib/ruby/gems/3.3.0/gems/webrick-1.9.2/sig/cgi.rbs
+#
+pipeline:
+  - name: "check the specified dir for file names"
+    runs: |
+      dir="${{inputs.dir}}"
+      name="${{inputs.name}}"
+      type="${{inputs.type}}"
+      files="${{inputs.files}}"
+
+      if [ ! -z "${files}" ] ; then
+        # NOOP: the other run mode was requested.
+        exit 0
+      fi
```

Comment:
```suggestion
```

We don't need this here i think.

Reply:
yes, otherwise you may get a bogus "PASS" message for this pipeline even if no `files` parameter was selected.